{
	"header": {
		"line": "CVE-2022-0168 (A denial of service (DOS) issue was found in the Linux kernel\\u2019s s ...)",
		"name": "CVE-2022-0168",
		"description": "(A denial of service (DOS) issue was found in the Linux kernel\\u2019s s ...)"
	},
	"annotations": [
		{
			"line": "\t{DSA-5127-1}",
			"type": "xref",
			"bugs": [
				"DSA-5127-1"
			]
		},
		{
			"line": "\t- linux 5.17.3-1",
			"type": "package",
			"package": "linux",
			"kind": "fixed",
			"version": "5.17.3-1"
		},
		{
			"line": "\t[buster] - linux <not-affected> (Vulnerable code not present)",
			"type": "package",
			"release": "buster",
			"package": "linux",
			"kind": "not-affected",
			"description": "Vulnerable code not present"
		},
		{
			"line": "\t[stretch] - linux <not-affected> (Vulnerable code not present)",
			"type": "package",
			"release": "stretch",
			"package": "linux",
			"kind": "not-affected",
			"description": "Vulnerable code not present"
		},
		{
			"line": "\tNOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2037386",
			"type": "NOTE",
			"description": "https://bugzilla.redhat.com/show_bug.cgi?id=2037386"
		}
	]
}
